docker-compose.yml

```diff
@@ -15,6 +15,8 @@ services:
     environment:
       LAALAA_API_HOST: "https://laa-legal-adviser-api-staging.apps.live-1.cloud-platform.service.justice.gov.uk"
       DJANGO_SETTINGS_MODULE: "fala.settings"
+      DB_USER: ${DB_USER:-postgres}
+      DB_HOST: ${DB_HOST:-db}
```

Comment:
This is useful syntax - I had to look up 'shell parameter expansions' to understand this.